docs/Reply.md

```diff
@@ -433,6 +435,13 @@ fastify.get('/botnet', async function (request, reply) {
 
 If you want to know more please review [Routes#async-await](Routes.md#async-await).
 
+
+<a name="send-reply-typed"></a>
+### .sendReplyTyped(code, data)
```

Comment:
I would prefer a more explicit `sendWithStatus(code, data)`

Reply:
The latest commit has renamed the method.